<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoicingSelect</class>
 <widget class="QWidget" name="VoicingSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>174</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Interpretation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="interpretBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Jazz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Literal</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Voicing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="voicingBox">
     <item>
      <property name="text">
       <string>Auto</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Root Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Close</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Drop 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Six Note</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Four Note</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Three Note</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="durationBox">
     <item>
      <property name="text">
       <string>Until next chord symbol</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stop at measure</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Segment duration</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
